### PR TITLE
Added RedisInsight v2.0 Browser Tool Tutorial

### PR DESCRIPTION
**Pull request by ajeetraina**

### CODE REVIEW

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+id: index-browser
+title: Visualize Redis database keys using RedisInsight Browser Tool
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
title: Visualize Redis Database keys using the RedisInsight Browser Tool
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+sidebar_label: Visualize Redis database keys using RedisInsight Browser Tool
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
sidebar_label: Visualize Redis Database keys using the RedisInsight Browser Tool 
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import RedisCard from '@site/src/theme/RedisCard';
+
+![My Image](images/image_redisinsight_browser.png)
+
+RedisInsight is a 100% free Redis GUI that allows you to visualise, monitor, and optimize while developing your applications with Redis. It provides an intuitive and efficient GUI for Redis allowing developers like you to interact with your databases and manage your data. RedisInsight v2.0 now incorporates a completely new tech stack based on the popular Electron and Elastic UI frameworks. You can run the application locally along with your favorite IDE, and it remains cross-platform, supported on Linux, Windows, and MacOS.
+
+
+
+## What's New in RedisInsight v2.0 Browser Tool?
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
## What's New in the RedisInsight v2.0 Browser Tool?
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+RedisInsight Browser lets you explore keys in your Redis server. You can add, edit and delete a key. You can even update the key expiry and copy the key name to be used in different places of the application. Below are the list of features available under the browser tool:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
RedisInsight Browser lets you explore keys in your Redis server. You can add, edit and delete a key. You can even update the key expiry and copy the key name to be used in different parts of the application. Below are the list of features available under the browser tool:
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+- Browse, filter and visualize key-value Redis data structures
+- Visual cues per data type
+- Quick view of size and ttl in the main browser view
+- Ability to filter by pattern and/or data type
+- Ability to change the number of keys to scan through during filtering
+- CRUD support for Lists, Hashes, Strings, Sets, Sorted Sets
+- Search within the data structure (except for Strings)
+- CRUD support for RedisJSON
+
+In order to understand the capabilities of the browser tool, let us take a simple example and demonstrate capabilities of each of browse tool options:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
In order to understand the capabilities of the browser tool, let us take a simple example and demonstrate each of the browser tool's options:
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+### Step 1. Download RedisInsight
+
+To use RedisInsight on a local Mac, you can download the software directly from the Redis official website:
+
+[Click this link](https://redis.com/redis-enterprise/redis-insight/#insight-form) to access a form that allows you to select the operating system of your choice.
+
+
+![My Image](images/image1.png)
+
+Fill out the rest of the form and click “Download.” Please note that the package name is the combination of the platform and version as shown here: redisinsight-preview-platform-version
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Fill out the rest of the form and click “Download”. Please note that the package name is the combination of the platform and version as shown here: redisinsight-preview-platform-version
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+![My Image](images/image31.png)
+
+Once clicked, it will redirect to [the link](https://redis.com/try-free/?utm_source=redis&utm_medium=app&utm_campaign=redisinsight) where you will need to complete the form.
+
+![My Image](images/image41.png)
+
+You can follow [this link](/create/rediscloud) to create a New Redis Enterprise Cloud database.
+In case you have an existing Redis database, follow the below steps.
+
+Assuming that you already have Redis database up and running locally, proceed to the next step to select "ADD REDIS DATABASE"
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Assuming that you already have Redis database up and running locally, proceed to the next step to select "ADD REDIS DATABASE".
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+![My Image](images/image_61.png)
+
+Once added, you will see the database added as shown below:
+
+![My Image](images/image_db.png)
+
+### Step 5: Open "Browser Tool"
+
+Click on "Key icon" on the left sidebar to open up the browser tool.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Click on the "Key" icon on the left sidebar to open up the browser tool.
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+
+![alt_text](images/image_browser.png "image_tooltip")
+
+
+### Step 6: Importing keys
+
+Let us import a user database( 6k keys). This dataset contains users stored as Redis Hash.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Let us import a user database (6k keys). This dataset contains users stored as Redis Hashes.
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+*   `last_name` : Last name.
+*   `email` : email address.
+*   `gender` : Gender (male/female).
+*   `ip_address` : IP address.
+*   `country` : Country Name.
+*   `country_code` : Country Code.
+*   `city` : City of the user.
+*   `longitude` : Longitude of the user.
+*   `latitude` : Latitude of the user.
+*   `last_login` : EPOC time of the last login.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
*   `last_login` : Epoch time of the last login.
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+
+### Step 7: Cloning the repository
+
+Open up the CLI terminal and run the below command:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Open up the CLI terminal and run the following command:
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+
+### Importing the user database:
+
+
+ ```bash
+  redis-cli -h localhost -p 6379 < ./import_users.redis
+ ```
+
+
+Refresh the keys database by clicking as shown below:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Refresh the keys view by clicking as shown below:
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+
+
+![alt_text](images/image_browser1.png "image_tooltip")
+
+
+You can get a real-time view of the data in your Redis database as shown below:
+
+
+
+Select any key in the key database and the results gets displayed in the right hand side that includes Fields and values.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
Select any key in the keys view and the key's value gets displayed in the right hand side that includes Fields and values.
```

File: docs/explore/redisinsightv2/browser/index-browser.mdx [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+
+Enter key name, field and value.
+
+
+
+
+
+### Step 9: Using CLI
+
+RedisInsight CLI lets you run commands against a redis server. You don’t need to remember the syntax - the integrated help shows you all the arguments and validates your command as you type.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/browser/index-browser.mdx]:
```suggestion
RedisInsight CLI lets you run commands against a Redis server. You don’t need to remember the syntax - the integrated help shows you all the arguments and validates your command as you type.
```

**Review by simonprickett — APPROVED** [review]
Made some small suggestions @ajeetraina

**Comment by ajeetraina** [comment]
> Made some small suggestions @ajeetraina

Thanks @simonprickett for being quick and helping with the review process.

**Pull request merged**